os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -126,6 +131,17 @@ static void rtl8730e_lcd_init(void)
 	sem_init(&g_next_frame_block, 0, 1);
 }
 
+static void rtl8730e_lcd_power_off(void)
+{
+	InterruptDis(lcdc_irq_info.num);
```

Comment:
I think we should add the defense logic we discussed in #6505, if isr is disabled, rtl8730e_lcd_put_area should just return to prevent stuck caused by timing issue, then we do not need #6505 anymore since isr controlled by lcd power.

Reply:
static int lcd_putarea(FAR struct lcd_dev_s *dev, fb_coord_t row_start, fb_coord_t row_end, fb_coord_t col_start, fb_coord_t col_end, FAR const uint8_t *buffer, fb_coord_t stride)
{
	FAR struct mipi_lcd_dev_s *priv = (FAR struct mipi_lcd_dev_s *)dev;
	while (sem_wait(&priv->sem) != OK) {
		ASSERT(get_errno() == EINTR);
	}
	if (priv->power == 0) {
		sem_post(&priv->sem);
		return ERROR;
	}

driver layer is already returning